style.css

```diff
@@ -0,0 +1,42 @@
+*{
+    font-family:Arial, Helvetica, sans-serif;
+    font-size: large;
+}
+.main{
+    text-align: center;
+    margin-top: 250px;
+    margin-left: 800px;
+}
+table{
```

Comment:
Spróbuj użyć czegoś bardziej współczesnego do ustawiania elementów na stronie. Flexbox / Grid to znacznie lepsze rozwiązania niż tabelki :D